packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts

```diff
@@ -329,8 +329,10 @@ export = {
 				await updateTags(id, newTags);
 				tags = await getWorkflowTags(id);
 			} catch (error) {
-				if (error instanceof QueryFailedError && error.message.includes('SQLITE_CONSTRAINT')) {
-					return res.status(404).json({ message: 'Some tags not found' });
+				if (error instanceof QueryFailedError) {
+					return res
+						.status(400)
+						.json({ message: 'Unable to perform update due to database error' });
```

Comment:
Since `QueryFailedError` could actually be some unhandled error, or communicating with the DB, can we change this to:

```suggestion
				if (error instanceof QueryFailedError && 'constraint' in error) {
					return res.status(404).json({ message: 'Some tags not found' });
```

In the long run we should perhaps update the drivers in typeorm to throw a `ConstraintError` instead.

Reply:
Hum good idea, especially now that we have our own typeorm fork